app/routes/chatbot.js

```diff
@@ -320,9 +320,9 @@ router.post('/', (req, res) => {
     })
     .catch(err => {
       if (err.message === 'broadcast declined') {
-        const msg = scope.broadcast.messages.prompt_declined;
+        const msg = scope.broadcast.msgDeclined;
         if (!msg) {
-          const logMsg = 'undefined broadcast.messages.prompt_declined';
+          const logMsg = 'undefined broadcast.msgDeclined';
```

Comment:
@DFurnes @weerd This relates to the thread we had in Slack today about naming convention, whether the various Gambit message properties should be named as `msgWhateverType` or `whateverTypeMessage`...